### PR TITLE
JAMES-1850 - Adopt the same top bar menu for the whole existing website

### PR DESCRIPTION
**Pull request by Laura-Royet**

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -63,6 +63,9 @@
 
         <links>
             <item
+                name="Home"
+                href="http://james.apache.org/index.html" />
```
Comment (chibenwa) [inline src/site/site.xml]: href is wanted to be a link to the Marketting website
(not a review remark, just explaination)

**Comment by chibenwa** [comment]
https://github.com/apache/james-jdkim/pull/4
https://github.com/linagora/james-jspf/pull/1
https://github.com/linagora/james-jsieve/pull/6

**Comment by Laura-Royet** [comment]
capture d ecran de 2016-10-28 14 07 15

**Comment by Laura-Royet** [comment]
https://github.com/linagora/james-mime4j/pull/1

**Pull request merged**